docs/getting-started.md

```diff
@@ -0,0 +1,88 @@
+# Getting Started
+
+## Installation
+
+Install `kitbag/router` with your favorite package manager
+
+```bash
+# bun
+bun add kitbag/router
+# yarn
+yarn add kitbag/router
+# npm
+npm install kitbag/router
+```
+
+## Define Basic Routes
+
+Create an array of possible routes. Learn more about [defining routes](/core-concepts/defining-routes).
+
+```js
+// /routes.ts
+import { Routes } from '@kitbag/router'
+
+const Home = { template: '<div>Home</div>' }
+const About = { template: '<div>About</div>' }
+
+export const routes = [
+  { name: 'home', path: '/', component: Home },
+  { name: 'path', path: '/about', component: About },
+] as const satisfies Routes 
+```
+
+## Plugin
+
+Create a router instance and pass it to the app as a plugin
+
+```js {2-3,6,9}
+import { createApp } from 'vue'
+import { createRouter } from '@kitbag/router'
+import { routes } from '/routes'
+import App from './App.vue'
+
+const router = createRouter(routes)
+const app = createApp(App)
+
+app.use(router)
+app.mount('#app')
+```
+
+## Update Registered Router
+
+This block utilizes [declaration merging](https://www.typescriptlang.org/docs/handbook/declaration-merging.html) to provide the internal types to match the actual router you're using. You put this in main.ts right after you call `createRouter`, or you can export your router and put this interface inside of a `router.d.ts` file, anywhere that your tsconfig can find it.
+
+```ts
+declare module '@kitbag/router' {
+  interface Register {
+    router: typeof router
+  }
+}
+```
+
+## RouterView
+
+Give your route components a place to be mounted
+
+```html {3-4}
+<div id="app">
+  ...
+  <!-- matched route.component gets rendered here -->
+  <router-view />
+</div>
+```
+
+This component can be mounted anywhere you want route components to be mounted. Nested routes can also have a nested `RouterView` which would be responsible for rendering any children that route may have. See more about [nested routes](/core-concepts/defining-routes#nested-routes).
+
+## RouterLink
+
+Use RouterLink for navigating between routes.
+
+```html {3-4}
+<div id="app">
+  ...
+  <!-- router-link renders as <a> with href -->
+  <router-link :to="{ route: 'home' }">Go somewhere</router-link>
+</div>
```

Comment:
Should we do example like this as templates? Like
```suggestion
<template>
  ...
  <!-- router-link renders as <a> with href -->
  <router-link :to="{ route: 'home' }">Go somewhere</router-link>
</template>
```

Reply:
I don't think so, at least not here. I think it's worth explaining to some developer that's new to the concept that this needs to be in root-level and while it's subtle, `<div class="app">` does convey more information than a generic `<template/>`

Reply:
I'm not following you here. What needs to be root level?

Reply:
oops, sorry I assumed this was the `<router-view/>` piece. I agree with you now that I actually read it more closely 😅